src/util.ts

```diff
@@ -6,7 +6,7 @@
 import { randomBytes } from 'crypto';
 
 function bytesToBase64url(s: Buffer): string {
-  return s.toString('base64').replace('+', '-').replace('/', '_');
+  return s.toString('base64').replace(/\+/g, 'Z').replace(/\//g, 'Z');
```

Comment:
Could we use `replaceAll` instead of introducing regular expressions here?

Reply:
I suppose you could. I didn't even know that existed. I did a quick search and found that it was added in ES2021. NodeJS didn't pick it up until v15. We are far passed that now. Since you're an enterprise company, it's entirely possible that you might break things for clients who have outdated NodeJS versions. Hopefully, they don't have anything older than 15, but you never know. Replace with a global regex works for everything.